search-app/src/cmr/search/results_handlers/results_handler_util.clj

```diff
@@ -1,65 +1,30 @@
 (ns cmr.search.results-handlers.results-handler-util
   "Provides useful functions for several of the result handlers."
   (:require
-   [clojure.set :as s]
    [cmr.common.util :as util]))
 
 (defn build-each-concept-association-list
   "This function builds a map with the passed in conept-key as the key and
   a list of concept ids from the passed in associations."
-  [concept-key associations for-concept-type]
-  (if (= :collection for-concept-type)
-    {concept-key (or (seq (util/remove-nils-empty-maps-seqs
-                           (mapv :variable-concept-id associations)))
-                     (seq (util/remove-nils-empty-maps-seqs
-                           (mapv :service-concept-id associations)))
-                     (seq (util/remove-nils-empty-maps-seqs
-                           (mapv :tool-concept-id associations)))
-                     (seq (util/remove-nils-empty-maps-seqs
-                           (mapv :concept-id associations))))}
-    {concept-key (or (seq (util/remove-nils-empty-maps-seqs
-                           (mapv :associated-concept-id associations)))
-                     (seq (util/remove-nils-empty-maps-seqs
-                           (mapv :concept-id associations))))}))
+  [concept-key associations]
+  ;;All the associations, generic or service/tool/variable associations,
+  ;;have been converted in indexer-app/src/cmr/indexer/data/concepts/association_util.clj
+  ;;to only contain concept-id and revision-id.
```

Comment:
Is this comment necesssary? An absolute path to the location of where conversion happens may drift out of sync with any future work as well.